### ISSUE TITLE
Server-side crash when attempting to sort chests

### ISSUE THREAD

**Issue by adam-the** [Chailotl/inventory-sort#7]
The mod list is included in the crash log, but if it helps, I am running the [Affogato modpack](https://modrinth.com/modpack/affogato).

[Crash Log](https://github.com/Chailotl/inventory-sort/files/15265140/crash.log.txt)

**Comment by adam-the** [comment]
Seems to be related to the FavoriteItems integration, the mod is present so I don't think it should be crashing? https://github.com/Chailotl/inventory-sort/blob/06b64ee25cab42ec87bc15571660ede2c5eb0edc/src/main/java/com/chailotl/inventory_sort/InventoryManager.java#L191

**Comment by Chailotl** [comment]
What the heck, it's in a try/catch, why is it crashing :sob:

I just tested right now on my own 1.20.1 modpack and it works fine, so I'll have to do some investigation with the linked Affogato modpack. By any chance was this on a dedicated server?

You are correct that the proper way is to use that `ExtensionItemStack` interface, but I'm using reflection to grab the method without needing any sort of dependency on ItemFavorites so that I can easily support other similar mods.

**Comment by adam-the** [comment]
> What the heck, it's in a try/catch, why is it crashing 😭
> 
> I just tested right now on my own 1.20.1 modpack and it works fine, so I'll have to do some investigation with the linked Affogato modpack. By any chance was this on a dedicated server?
> 
> You are correct that the proper way is to use that `ExtensionItemStack` interface, but I'm using reflection to grab the method without needing any sort of dependency on ItemFavorites so that I can easily support other similar mods.

Yeah this was a dedicated server. I just had a thought - could it be related to the server being ran through GraalVM? I will test with a regular JVM.

**Comment by pajicadvance** [comment]
I think it's crashing because you're loading a client-side class that doesn't exist on the dedicated server. This wouldn't only cause a crash in my modpack, but basically anywhere.

https://github.com/Chailotl/inventory-sort/blob/06b64ee25cab42ec87bc15571660ede2c5eb0edc/src/main/java/com/chailotl/inventory_sort/InventoryManager.java#L187

The re-mapped stack trace says that the mod is trying to load the Screen class, which obviously doesn't exist server-side:

```
java.lang.NoClassDefFoundError: net/minecraft/Screen
        at com.chailotl.inventory_sort.InventoryManager.isFavorite(InventoryManager.java:187)
        at com.chailotl.inventory_sort.InventoryManager.sortInventory(InventoryManager.java:33)
        at com.chailotl.inventory_sort.Main.lambda$onInitialize$2(Main.java:30)
        at net.minecraft.server.ServerTask.run(ServerTask.java:18)
        at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:156)
        at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:23)
        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:782)
        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:164)
        at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:130)
        at net.minecraft.server.MinecraftServer.runOneTask(MinecraftServer.java:14818)
        at net.minecraft.server.MinecraftServer.runTask(MinecraftServer.java:758)
        at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:115)
        at net.minecraft.server.MinecraftServer.runTasksTillTickEnd(MinecraftServer.java:742)
        at net.minecraft.server.MinecraftServer.handler$bgh000$carpet$modifiedRunLoop(MinecraftServer.java:16451)
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:650)
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.gui.screen.Screen
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
        at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
        at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
        ... 17 more
```

**Comment by adam-the** [comment]
fixed by #8

**Comment by Chailotl** [comment]
That `IS_ITEM_FAVORITES_LOADED` constant was supposed to be in `Main.class`, not `ClientMain.class`, I feel so silly 🤦

The proper solution is to move it there, as it does not need to exist on both sides.
